build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
```

Comment:
Do we still need this? I assume that now that it is declared in `settings.gradle.kts` it shouldn't be necessary.

Reply:
You're right, we don't need! Removed.